library/index.ts

```diff
@@ -7,3 +8,17 @@ const shouldEnable = shouldEnableFirewall();
 if (supported && shouldEnable) {
   require("./agent/protect").protect();
 }
+
+import { setUser } from "./agent/context/user";
+import { setupExpressIntegration } from "./integrations/express";
+import { shouldBlockRequest } from "./integrations/shouldBlockRequest";
+import { setupHonoIntegration } from "./integrations/hono";
+import { setupHapiIntegration } from "./integrations/hapi";
+
+export {
+  setUser,
+  shouldBlockRequest,
+  setupExpressIntegration,
+  setupHonoIntegration,
+  setupHapiIntegration,
+};
```

Comment:
So on Deno it will crash but only after logging the message, right?

Reply:
With new Deno v2:
```
error: Uncaught (in promise) ReferenceError: exports is not defined
Object.defineProperty(exports, "__esModule", { value: true });
                      ^
    at file:///Users/timokoessler/Git/firewall-node/build/index.js:2:23

    info: Deno supports CommonJS modules in .cjs files, or when there's a package.json
          with "type": "commonjs" option and --unstable-detect-cjs flag is used.
    hint: Rewrite this module to ESM,
          or change the file extension to .cjs,
          or add package.json next to the file with "type": "commonjs" option
          and pass --unstable-detect-cjs flag.
    hint: See https://docs.deno.com/go/commonjs for details
```